src/MicrosoftTeams.schema.json

```diff
@@ -173,15 +172,85 @@
                         "description": "A unique identifier for the bot that matches its ID in the bot framework.",
                         "maxLength": 64
                     },
+                    "needsChannelSelector": {
+                        "type": "boolean",
+                        "description": "This value describes whether or not the bot utilizes a user hint to add the bot to a specific channel.",
```

Comment:
I don't think this is something that we can express in the schema but we could add it to the property description as documentation for developers.